server_admin/topics/user-federation/ldap.adoc

```diff
@@ -32,6 +32,13 @@ The benefit to this approach is that you do not have the overhead of importing a
 
 This storage mode is controled by the `Import Users` switch.  Set to `On` to import users.
 
+NOTE: In case that import of users is disabled, there is no support to save user profile attributes and most of other user metadata
+      into the {project_name} database. The only exception are user profile metadata, which are mapped to the LDAP. This possibly includes
+      role mappings, group mappings and other metadata based on the configuration of your LDAP mappers.
+      When the attempt is made to change some non-LDAP mapped user data, the update of the user will not be possible. For example
```

Comment:
```s/to change some non-LDAP mapped/some of the non-LDAP mapped/g```

Reply:
I believe you meant:
`s/to change some non-LDAP mapped/to change some of the non-LDAP mapped/g` ?

I changed it that way.

Reply:
Yes, it was just about adding `of` word (sorry for typo in the copy-pasting). You got it right.